docs/v1/P4Runtime-Spec.mdk

```diff
@@ -4729,7 +4729,7 @@ defined as follows:
 ~ Begin Proto
 message WriteRequest {
   uint64 device_id = 1;
-  uint64 role = 2;
+  string role = 2;
```

Comment:
The number two needs to be 6 too...😉

Reply:
It's a good question of whether we want to keep the protos in the spec completely in sync with the real protos. As a middle ground for now, let me update the ids, but not add the deprecated fields in the spec (even though they are in the proto). I don't feel strongly about this though, happy to add the deprecated fields as well.